ui/packages/catalog-ui-search/src/main/webapp/component/search-form-interactions/search-form-interactions.view.js

```diff
@@ -13,14 +13,14 @@
  *
  **/
 /*global define, window*/
-var wreqr = require('wreqr');
-var Marionette = require('marionette');
-var template = require('./search-form-interactions.hbs');
-var CustomElements = require('js/CustomElements');
-var user = require('component/singletons/user-instance');
-var LoadingView = require('component/loading/loading.view');
-var announcement = require('component/announcement');
-var ConfirmationView = require('component/confirmation/confirmation.view');
+let wreqr = require('wreqr');
```

Comment:
Not to leave unnecessary nitpicks, but believe imports should be `const` to remain consistent with other files following ES6 syntax for imports.

Reply:
done